groupbalancer.go

```diff
@@ -185,3 +368,93 @@ func findGroupBalancer(protocolName string, balancers []GroupBalancer) (GroupBal
 	}
 	return nil, false
 }
+
+// findRack is the default rack resolver strategy.  It currently only supports
+//  * ECS with the task metadata endpoint enabled (returns the container
+//    instance's availability zone)
+//  * Linux EC2 (returns the instances' availability zone)
```

Comment:
super minor nit:  `instances'` -> `instance's`?

Reply:
thanks as always for proofreading. 😄